libfwupdplugin/fu-common-version.c

```diff
@@ -235,6 +235,73 @@ _g_ascii_is_digits(const gchar *str)
 	return TRUE;
 }
 
+static guint
+_fwupd_version_format_sections(FwupdVersionFormat fmt)
```

Comment:
nit; it's not really format sections - it's count the "number" of sections

Reply:
_fwupd_version_format_number_sections perhaps?

Reply:
thats good